### PR TITLE
logstash 5.6.3

### PR DESCRIPTION
**Pull request by jasontedor**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ilovezfs** [comment]
```
Error: inreplace failed
bin/logstash:
  expected replacement of /^\. "\$\(cd `dirname \$SOURCEPATH`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"/ with ". /usr/local/Cellar/logstash/5.6.3/libexec/bin/logstash.lib.sh"
```

**Comment by jasontedor** [comment]
Thanks I will address this later today.

**Comment by ilovezfs** [comment]
@jasontedor any update here?

**Comment by jasontedor** [comment]
Sorry for the delay here, I pushed a fix.

**Comment by jasontedor** [comment]
These builds are running on JDK 9 which is not supported by Logstash. What can be done here?

**Comment by ilovezfs** [comment]
See https://github.com/Homebrew/homebrew-core/commit/2fb3a914ef471bd9b405b1a94a5f5f52d12c3202

**Comment by jasontedor** [comment]
Thanks, I pushed a commit for that.

File: Formula/logstash.rb [hunk 1]
```diff
@@ -19,12 +19,13 @@ def install
       cd "tar"
     end
 
-    inreplace %w[bin/logstash], %r{^\. "\$\(cd `dirname \$SOURCEPATH`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"}, ". #{libexec}/bin/logstash.lib.sh"
+    inreplace %w[bin/logstash], %r{^\. "\$\(cd `dirname \${SOURCEPATH}`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"}, ". #{libexec}/bin/logstash.lib.sh"
     inreplace %w[bin/logstash-plugin], %r{^\. "\$\(cd `dirname \$0`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"}, ". #{libexec}/bin/logstash.lib.sh"
     inreplace %w[bin/logstash.lib.sh], /^LOGSTASH_HOME=.*$/, "LOGSTASH_HOME=#{libexec}"
     libexec.install Dir["*"]
     bin.install_symlink libexec/"bin/logstash"
```
Comment (ilovezfs) [inline Formula/logstash.rb]: You'll want to replace the two `bin.install_symlink libexec/"bin/logstash"` and `bin.install_symlink libexec/"bin/logstash-plugin"` with

```
bin.install libexec/"bin/logstash", libexec/"bin/logstash-plugin"
```
Then bin.env_script_all_files puts them back in libexec/bin (where they started!) and replaces them with wrapper scripts.

Comment (jasontedor) [reply Formula/logstash.rb]: Thanks, I pushed a commit for that.

**Comment by jasontedor** [comment]
The build passed on Sierra and High Sierra but failed on El Capitan. This appears to be an infrastructure issue:

```
18:20:08 + brew update
18:20:08 /var/folders/kw/j7f63hdn1lj39702nh90xm9w0000gp/T/jenkins8303859000565163637.sh: line 3: brew: command not found
```

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
Thanks @jasontedor! :ship:'d at last!

**Comment by jasontedor** [comment]
Awesome. Thank you! Again, sorry for the delay and thanks for your help.

**Comment by ilovezfs** [comment]
No worries, and you're welcome! Are you planning to open the 5.6.4 PR now?

**Pull request merged**